packages/dicomImageLoader/package.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "@cornerstonejs/dicom-image-loader",
+  "version": "0.0.1",
+  "description": "Cornerstone Image Loader for DICOM WADO-URI and WADO-RS and Local file",
+  "keywords": [
+    "DICOM",
+    "WADO",
+    "cornerstone",
+    "medical",
+    "imaging"
+  ],
+  "author": "@cornerstonejs (previously Chris Hafey)",
+  "homepage": "https://github.com/cornerstonejs/cornerstone3D-beta",
+  "license": "MIT",
+  "main": "./dist/cornerstoneDICOMImageLoader.bundle.min.js",
```

Comment:
Suggest using main as the src/index.ts file, and use exports to define the exported files.  Please ensure you export an unobfuscated version too.

Reply:
I will handle build later